medusa/clients/torrent/utorrent_client.py

```diff
@@ -52,13 +52,14 @@ def _request(self, method='get', params=None, data=None, files=None, cookies=Non
     def _get_auth(self):
 
         self.response = self.session.get(urljoin(self.url, 'token.html'), verify=False)
-        self.auth = re.findall('<div.*?>(.*?)</', self.response.text)[0]
-
-        return self.auth if not self.response.status_code == 404 else None
+        if not self.response.status_code == 404:
+            self.auth = re.findall('<div.*?>(.*?)</', self.response.text)[0]
+            return self.auth
 
     def _add_torrent_uri(self, result):
         return self._request(params={
             'action': 'add-url',
+            # requests (?) limits the param length to 1024 chars
             's': result.url[:1024],
```

Comment:
I don't think there will be a better solution to this. And this should be fixed in utorrent.

But If I understand this correct it will leave invalid (malformed) tracker url's.
To prevent that, we could use this:

```python
 '&'.join(param[:1024].split('&')[:-1]) if param[:1024] < param else param
```
What do you think?